packages/nimble-components/src/table/specs/table-columns-hld.md

```diff
@@ -291,61 +323,10 @@ const numberWithUnitCellView = NumberWithUnitCellView.compose({
 });
 ```
 
-Here we have a cell visualized in different ways based on custom logic:
```

Comment:
Didn't feel like we really need this example anymore. Visualizing custom logic and having custom state is no longer interesting / you just implement a normal fast component. The only interesting bit is responding to columnConfig and cellRecord state which feels covered in other examples.